### DISCUSSION TITLE
r/china-travel-layoff-trip-summary-and-recommendation

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#85] [Announcements]
# r/china-travel-layoff-trip-summary-and-recommendation

总结去过的所有景点，把最好玩儿的推荐给你

https://blog.ops-coffee.cn/r/china-travel-layoff-trip-summary-and-recommendation

**Comment by fcatat** [comment]
“向往的生活”

**Comment by ddonano** [comment]
羡慕啊

**Comment by annilq** [comment]
真不错，看来换个车平时出去玩能节省不少钱
